admiral/common_functions.sqf

```diff
@@ -172,6 +184,27 @@ adm_common_fnc_getAliveUnits = {
     _aliveUnits;
 };
 
+adm_common_fnc_getAliveSideUnits = {
+    FUN_ARGS_2(_groupsArray,_side);
+    
+    DECLARE(_aliveSideUnits) = [];
+    {
+        DECLARE(_groups) = _x;
+        {
+            DECLARE(_groupUnits) = units _x;
+            FILTER_PUSH_ALL(_aliveSideUnits,_groupUnits,{(alive _x) && (side _x == _side)});
```

Comment:
No need for parens for the first expression in the condition. Use short-circuit syntax for multiple expressions in bool expression.